tee-worker/litentry/core/assertion-build/src/a4.rs

```diff
@@ -106,57 +109,48 @@ pub fn build(req: &AssertionBuildRequest, min_balance: ParameterString) -> Resul
 	//    value: true
 	// ]
 	for (network, addresses) in identities {
+		if network != Web3Network::Ethereum
```

Comment:
This should be already filtered out by https://github.com/litentry/litentry-parachain/blob/6da97441101348d8b1bfc486087514fd2c57088c/tee-worker/app-libs/stf/src/trusted_call_litentry.rs#L138-L151 as @Kailai-Wang mentioned. 

Is there any reason you kept it?